sdk/maps/maps-timezone-rest/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "@azure-rest/maps-timezone",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "A generated SDK for MapsTimezoneClient.",
+  "keywords": ["node", "azure", "cloud", "typescript", "browser", "isomorphic"],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/maps-timezone-rest.d.ts",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": { "url": "https://github.com/Azure/azure-sdk-for-js/issues" },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/maps-timezone-rest.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "engines": { "node": ">=18.0.0" },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "echo skipped.",
+    "build:test": "tsc -p . && dev-tool run bundle",
+    "build:debug": "tsc -p . && dev-tool run bundle && api-extractor run --local",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-browser dist-esm test-dist temp types *.tgz *.log",
+    "execute:samples": "echo skipped",
+    "extract-api": "rimraf review && mkdirp ./review && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript swagger/README.md && npm run format",
+    "integration-test:browser": "dev-tool run test:browser",
+    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 5000000 'dist-esm/test/**/*.spec.js'",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json api-extractor.json src test --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json api-extractor.json src test",
+    "pack": "npm pack 2>&1",
+    "test:browser": "npm run clean && npm run build:test && npm run unit-test:browser",
+    "test:node": "npm run clean && npm run build:test && npm run unit-test:node",
+    "test": "npm run clean && npm run build:test && npm run unit-test",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "unit-test:node": "dev-tool run test:node-ts-input -- --timeout 1200000 --exclude 'test/**/browser/*.spec.ts' 'test/**/*.spec.ts'",
+    "unit-test:browser": "dev-tool run test:browser",
```

Comment:
![image](https://github.com/user-attachments/assets/5532abea-7ca8-463e-96f7-8a3b98ce5171)
browser tests job in the pipeline is failing because all the tests are being skipped in playback, and karma framework thinks that's a failure.
```suggestion
    "unit-test:browser": "echo skipped",
```